data/journal_check/bug_refs.json

```diff
@@ -0,0 +1,82 @@
+{
+    "bsc#1022527": {
+        "description": ".*wickedd.*ni_process_reap.*blocking waitpid.*",
+        "products": ["sle-micro", "opensuse", "microos", "sle"],
+        "type": "feature"
+    },
+    "bsc#000000": {
```

Comment:
That's an invalid bug ref; if we tag stuff soft-fail per code, it should point to something existing.

Reply:
Agreed, however it is there for 2 years without any objections. https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7052

Reply:
a bad argument to not change/address it

as there is not even a bug report, there is certainly no reason to expect it ever to be fixed. As such one could just as well not mark it as a softfail/bug

Question of course now if anybody even remembers what exactly the bug is

Reply:
Ofc, far from a good argument and I have honestly expected this kind of a reply. :)
Unfortunately, I have no clue what is the real status, maybe @Vogtinator or @jlausuch might have more information. If there are no objections, it can be labelled as a message to be ignored, at least for microos and slem.

Reply:
I don't remember cause I started in the project when this was already there. But could it be because we are forcing a transactional update with a broken package on purpose?
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/transactional/transactional_update.pm#L95

If this is the reason, I would vote for `type: ignore`

Reply:
Ok, set to ignore as of now

Reply:
Yeah, those messages are expected and need to be there